metapackages/plugins-node-all/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@opentelemetry/plugins-node-all",
+  "version": "0.10.2",
+  "description": "Metapackage which bundles opentelemetry node core and contrib plugins",
+  "author": "OpenTelemetry Authors",
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js#readme",
+  "license": "Apache-2.0",
+  "publishConfig": {
+      "access": "public"
+  },
+  "repository": {
+      "type": "git",
+      "url": "git+https://github.com/open-telemetry/opentelemetry-js.git"
+  },
+  "bugs": {
+      "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
+  },
+  "dependencies": {
+      "@opentelemetry/plugin-express": "0.9.0",
+      "@opentelemetry/plugin-ioredis": "0.9.0",
+      "@opentelemetry/plugin-mongodb": "0.9.0",
+      "@opentelemetry/plugin-mysql": "0.9.0",
+      "@opentelemetry/plugin-pg": "0.9.0",
+      "@opentelemetry/plugin-pg-pool": "0.9.0",
+      "@opentelemetry/plugin-redis": "0.9.0",
+      "@opentelemetry/plugins-node-core": "^0.10.2"
```

Comment:
i think we can bump this to `0.11.0`

Reply:
@michaelgoin can you resolve this then we can merge?

Reply:
updated.